policy/release_manager.md

```diff
@@ -0,0 +1,49 @@
+# Release manager role
+
+## Appointment
+
+A release manager (RM) is appointed for a specific upcoming release.
```

Comment:
```suggestion
A release manager (RM) is appointed for a specific upcoming minor or major release, i.e. `x.y.z` -> "x.y+1.0` or `x.y.z` -> `x+1.0.0`.
```

Reply:
Makes sense. I wanted to stay away from it, because of possible changes after `1.0`, but I think in this way it just works.